packages/loopring_v3/contracts/impl/libtransactions/TransferTransaction.sol

```diff
@@ -0,0 +1,129 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2017 Loopring Technology Limited.
+pragma solidity ^0.6.10;
+pragma experimental ABIEncoderV2;
+
+import "../../iface/ExchangeData.sol";
+import "../../thirdparty/BytesUtil.sol";
+import "../../lib/EIP712.sol";
+import "../../lib/FloatUtil.sol";
+import "../../lib/MathUint.sol";
+import "../../lib/SignatureUtil.sol";
+
+
+/// @title TransferTransaction
+/// @author Brecht Devos - <brecht@loopring.org>
+library TransferTransaction
+{
+    using BytesUtil            for bytes;
+    using FloatUtil            for uint;
+    using MathUint             for uint;
+    using SignatureUtil        for bytes32;
+
+    bytes32 constant public TRANSFER_TYPEHASH = keccak256(
+        "Transfer(address from,address to,uint16 tokenID,uint256 amount,uint16 feeTokenID,uint256 fee,uint256 data,uint32 nonce)"
+    );
+
+    /*event ConditionalTransferConsumed(
```

Comment:
Makes things a bit more expensive for all these transactions and it doesn't really make much sense to do it I think. We don't log any events for most transactions when authorized with an EdDSA signature, so why do it because they happen to use an ECDSA signature? To know what's going on the block data needs to be processed for all transactions any way. But I used these for testing and was undecided what to do with them.

Reply:
I asked this because we emit the DepositProcessed and ForcedWithdrawalProcessed, shall we also disable those two events?

Reply:
Sure, fine with me.